client/lib/plans/constants.js

```diff
@@ -477,6 +479,12 @@ export const FEATURES_LIST = {
 		)
 	},
 
+	[ FEATURE_UPLOAD_PLUGINS ]: {
+		getSlug: () => FEATURE_UPLOAD_PLUGINS,
+		getTitle: () => i18n.translate( 'Upload plugins' ),
```

Comment:
Let's change this to `Install plugins`, and the description to `Install custom plugins on your site`.